docs/mine/lotus/split-markets-miners.md

```diff
@@ -122,29 +173,45 @@ lotus-miner --markets-repo=~/.lotusmarkets init service --type=markets \
                                                         /tmp/backup.cbor
 ```
 
-3. Optionally update your miner's `multiaddr` on-chain - in case your `markets` instance is publicly exposed at a different location compared to your existing monolith miner, you also need to update your `multiaddr` on-chain and advertise the correct one to clients:
+3. Optionally update your miner's `multiaddr` on-chain - in case your `markets`
+instance is publicly exposed at a different location compared to your existing
+monolith miner, you also need to update your `multiaddr` on-chain to advertise
+the correct address to clients:
 
 ```shell
 lotus-miner actor set-addrs <NEW_MULTIADDR>
 ```
 
-### 4. Move the DAG store directory to the markets node repository
+#### 4. Move the DAG store directory to the markets node repository
+
+If you are running a lotus version with a DAG store, you can optionally move the
+DAG store directory to the lotus markets repository, to avoid having to reindex
+all storage deals when the markets node starts up.
 
 ```shell
 mv ~/.lotusminer/dagStore ~/.lotusmarkets/
 ```
```

Comment:
This needs to be optional, if the user had started 1.11.2-rc or m1.x with a monolith deployment.

Reply:
Are you saying we should add that phrasing here?